ArduPlane/servos.cpp

```diff
@@ -442,9 +442,10 @@ void Plane::set_servos_controlled(void)
         } else if (g.throttle_passthru_stabilize) {
             // manual pass through of throttle while in FBWA or
             // STABILIZE mode with THR_PASS_STAB set
-            SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, channel_throttle->get_control_in_zero_dz());
+            SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, get_throttle_input(true));
```

Comment:
My understanding is that this is a wrong change? I thought `throttle_passthru_stabilize` meant direct throttle output from the input, without any manipulation? (I've never used it though, this is just my understanding based upon the dev call and the parameter description.

Reply:
this puts it inline with what is done in manual, https://github.com/ArduPilot/ardupilot/blob/a8a71db25871003fc7fcd06b43cba16d808ad94e/ArduPlane/servos.cpp#L329 

the only difference is get_throttle_input(true) could be negative if reverse thrust is active
https://github.com/ArduPilot/ardupilot/blob/a8a71db25871003fc7fcd06b43cba16d808ad94e/ArduPlane/reverse_thrust.cpp#L107